cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1468,10 +1468,21 @@
   </property>
 
   <property>
+    <name>security.authorization.authorizer.extension.jar.path</name>
+    <value></value>
+    <description>
+      If an external authorization system is used for authorizing operations on CDAP entities, this property sets
+      the path to the bundled JAR file containing the extension source code. This property is only used when
```

Comment:
Saying `This jar is only used when` might be clearer than saying  `This property is only used when`.